### PR TITLE
Add `meta-description`

### PR DESCRIPTION
**Pull request by Mikaelsu**
And other updates prompted by our SEO agency

### CODE REVIEW

**Comment by Mikaelsu** [comment]
Removed the language check to keep keywords

**Comment by jouni** [comment]
There's no support for a `meta-description` frontmatter attribute. Only `description`, which does end up in the DOM as meta tag, but also as a visible page item under the page heading.

**Comment by Mikaelsu** [comment]
I'll rework this PR next month.

File: articles/components/grid/index.asciidoc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Grid
 tab-title: Usage
 layout: tabbed-page
+meta-description: The Vaadin Grid component is accessible, themable, and enterprise-grade.
```
Comment (jouni) [inline articles/components/grid/index.asciidoc:5]: This is a bit brief and not very informative. I would expect the words "data grid" or "tabular data" to appear somewhere.

File: articles/components/combo-box/index.asciidoc [hunk 2]
```diff
@@ -2,6 +2,7 @@
 tab-title: Usage
 layout: tabbed-page
 title: Combo Box
+meta-description: The Vaadin ComboxBox component is accessible, themable, and enterprise-grade. Made for Java and Typescript, use it with Flow or a web frontend of your choice.
```
Comment (jouni) [inline articles/components/combo-box/index.asciidoc:5]: This doesn't say what the component does. But perhaps "Combo Box" is well known, and we don't need to be patronizing?

**Review by jouni — COMMENTED** [review]
@Mikaelsu, are you still working on this?

**Pull request closed without merging**